From bff654d7e39ead46cb007c2312e702edfd79d2eb Mon Sep 17 00:00:00 2001
From: Aditya Garg <gargaditya08@live.com>
Date: Tue, 12 Sep 2023 10:47:52 +0530
Subject: [PATCH] Change group for Debain based distros

---
 src/main.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main.rs b/src/main.rs
index fe5f825..86304ef 100644
--- a/src/main.rs
+++ b/src/main.rs
@@ -279,7 +279,7 @@ fn main() {
 
     PrivDrop::default()
         .user("nobody")
-        .group("nobody")
+        .group("nogroup")
         .group_list(&groups)
         .apply()
         .unwrap_or_else(|e| { panic!("Failed to drop privileges: {}", e) });
-- 
2.39.2

